### DISCUSSION TITLE
feature request :  option to disable SSL verification globaly

### DISCUSSION THREAD

**Discussion by dgoosens** [darrenburns/posting#141] [Ideas]
hi,

As stated in the documentation:

> SSL verification can be disabled on a per-request basis in the "Options" tab.

Would it be feasible to add a configuration option somehow to simply disable SSL verification completely ?

Thanks  
Dmitri

**Comment by johnend** [comment]
Agreed this would be a good feature to include. Maybe have it configurable by env, or collection based. Or just have the toggle be global rather than per request.
